lib/Forgotpassword.dart

```diff
@@ -0,0 +1,248 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:signup_module/main.dart';
+
+class MyforgotPasspage extends StatefulWidget {
+  const MyforgotPasspage({Key? key}) : super(key: key);
+
+  @override
+  _MyforgotPasspageState createState() => _MyforgotPasspageState();
+}
+
+class _MyforgotPasspageState extends State<MyforgotPasspage> {
+  final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+  TextEditingController controller = TextEditingController();
+  bool eye = true;
+  bool ey = true;
+
+  @override
+  void initState() {
+    // TODO: implement initState
+    super.initState();
+    controller.addListener(() {});
+  }
+
+  void va() {
+    setState(() {
+      ey = !ey;
+    });
+  }
+
+  void vaw() {
+    setState(() {
+      eye = !eye;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Stack(
+        alignment: Alignment.center,
+        children: [
+          Container(
+            height: MediaQuery.of(context).size.height * 1,
+            width: double.infinity,
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage("asset/image/background_image.jpeg"),
+                fit: BoxFit.cover,
+              ),
+            ),
+          ),
+          ClipRRect(
+            borderRadius: BorderRadius.circular(25),
+            child: BackdropFilter(
+              filter: ImageFilter.blur(sigmaX: 100.0, sigmaY: 100.0),
+              child: SizedBox(
+                height: MediaQuery.of(context).size.height * .7,
+                width: MediaQuery.of(context).size.width * .9,
+                child: Form(
+                  key: formKey,
+                  child: SingleChildScrollView(
+
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        SizedBox(
+                          height: 70,
+                        ),
+                        Text(
+                          "Forgot Password",
+                          style: TextStyle(
+                              color: Colors.white,
+                              fontSize: 30,
+                              fontFamily: "san ferif",
+                              fontWeight: FontWeight.bold),
+                        ),
+                        SizedBox(
+                          height: 70,
```

Comment:
Do not use coma at last whenever there are only two parameters, Unless it breaks the 80 character rule.